redpanda/values.yaml

```diff
@@ -160,7 +160,19 @@ statefulset:
   resources:
     limits:
       cpu: 1
-      memory: 2Gi
```

Comment:
I believe this could be removed due to the associated lines being removed in `_helpers.tpl`

Reply:
When I revert this change I see the following error as cgroup has less memory than combining:
```
--memory:2G
--reserve-memory=400k
```

an error:
```
exec /usr/bin/rpk redpanda start --smp=1 --memory=2G --reserve-memory=400k --advertise-kafka-addr=kafka://redpanda-0.redpanda.redpanda.svc.cluster.local.:9092,external://redpanda-0.streaming.rockdata.io:32005, --kafka-addr=kafka://10.244.0.10:9092,external://10.244.0.10:9093, --advertise-rpc-addr=redpanda-0.redpanda.redpanda.svc.cluster.local.:33145 --rpc-addr=10.244.0.10:33145 --schema-registry-addr=internal://10.244.0.10:8081,external://0.0.0.0:18081 --advertise-pandaproxy-addr=internal://redpanda-0.redpanda.redpanda.svc.cluster.local.:8082,external://172.18.0.2:8083 --pandaproxy-addr=internal://10.244.0.10:8082,external://10.244.0.10:8083 --default-log-level=info
...
Could not initialize seastar: std::runtime_error (insufficient physical memory: needed 2147483648 available 2147074048)
```